WeakAurasTemplates/TriggerTemplatesDataMists.lua

```diff
@@ -187,21 +186,107 @@ templates.class.PALADIN = {
+        { spell = 20271, type = "ability", requiresTarget = true, usable = true }, -- Judgment
+        { spell = 20473, type = "ability" }, -- Holy Shock
+        { spell = 20925, type = "ability", buff = true, talent = 9 }, -- Sacred Shield
+        { spell = 24275, type = "ability", overlayGlow = true, requiresTarget = true, usable = true }, -- Hammer of Wrath
+        { spell = 25780, type = "ability", buff = true }, -- Righteous Fury
+        { spell = 26573, type = "ability", totem = true }, -- Consecration
+        { spell = 31821, type = "ability", buff = true }, -- Devotion Aura
+        { spell = 31842, type = "ability", buff = true }, -- Divine Favor
+        { spell = 31850, type = "ability", buff = true }, -- Ardent Defender
+        { spell = 31884, type = "ability", buff = true }, -- Avenging Wrath
+        { spell = 31935, type = "ability", debuff = true, usable = true }, -- Avenger's Shield
+        { spell = 35395, type = "ability", requiresTarget = true, usable = true }, -- Crusader Strike
+        { spell = 53563, type = "ability", buff = true }, -- Beacon of Light
+        { spell = 53595, type = "ability", requiresTarget = true, usable = true }, -- Hammer of the Righteous
+        { spell = 53600, type = "ability", requiresTarget = true, usable = true }, -- Shield of the Righteous
+        { spell = 54428, type = "ability", buff = true }, -- Divine Plea
+        { spell = 62124, type = "ability", debuff = true, requiresTarget = true }, -- Reckoning
+        { spell = 85256, type = "ability", overlayGlow = true, requiresTarget = true }, -- Templar's Verdict
+        { spell = 85499, type = "ability", buff = true, talent = 1 }, -- Speed of Light
+        { spell = 85673, type = "ability", overlayGlow = true }, -- Word of Glory
+        { spell = 86659, type = "ability", buff = true }, -- Guardian of Ancient Kings
+        { spell = 86669, type = "ability", buff = true }, -- Guardian of Ancient Kings
+        { spell = 86698, type = "ability", buff = true }, -- Guardian of Ancient Kings
+        { spell = 96231, type = "ability", requiresTarget = true }, -- Rebuke
+        { spell = 105593, type = "ability", debuff = true, requiresTarget = true, talent = 4 }, -- Fist of Justice
+        { spell = 105809, type = "ability", buff = true, talent = 13 }, -- Holy Avenger
+        { spell = 114039, type = "ability", buff = true, talent = 10 }, -- Hand of Purity
+        { spell = 114157, type = "ability", talent = 18 }, -- Execution Sentence
+        { spell = 114158, type = "ability", totem = true, talent = 17 }, -- Light's Hammer
+        { spell = 114163, type = "ability", buff = true, talent = 8 }, -- Eternal Flame
+        { spell = 114165, type = "ability", talent = 16 }, -- Holy Prism
+        { spell = 115750, type = "ability" }, -- Blinding Light
+        { spell = 119072, type = "ability" }, -- Holy Wrath
+        { spell = 145067, type = "ability" }, -- Turn Evil
```

Comment:
duplicate of 10326

Reply:
This is the version of Turn Evil when talented into Evil is a Point of View. The other is without the talent.